### PR TITLE
New About Page

### PR DESCRIPTION
**Pull request by mab6bf**
- Implements new About page
- Does not yet include the "View Agency Quarterly Submission Status" button, as the Quarterly Submission Status page hasn't been built yet

Mock: https://projects.invisionapp.com/d/main#/console/8586929/262682080/preview
Story: https://federal-spending-transparency.atlassian.net/browse/DS-1944

- [x] Code Review
- [x] Design Review
- [x] Content Approval

### CODE REVIEW

File: src/_scss/pages/about/_section.scss [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                    display: block;
+                    font-weight: $font-semibold;
+                }
+            }
+        }
+
+        button {
+            display: block;
+            margin: rem(19) 0 rem(38);
+            padding: rem(16) rem(32) rem(13);
```
Comment (kevinli-work) [inline src/_scss/pages/about/_section.scss]: Any reason why the button top padding is different from the bottom padding? It results in non-vertically centered text.

File: src/_scss/pages/about/_contentWrap.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    color: $color-base;
+    border-top: 1px solid $color-gray-border;
+    border-right: 1px solid $color-gray-border;
+    border-bottom: 1px solid $color-gray-border;
+
+    .about-padded-content {
+        padding: rem(35);
+    }
+
+    @media(min-width: $large-screen) {
```
Comment (kevinli-work) [inline src/_scss/pages/about/_contentWrap.scss]: Please use the standard media mixin we're using everywhere else on the site `@include media($screen-var)`

File: src/_scss/pages/about/header/header.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+@import "components/pageTitleBar/pageTitleBar";
```
Comment (kevinli-work) [inline src/_scss/pages/about/header/header.scss]: Why do this? Why not just do this import in the parent file and delete this file?

File: src/js/components/about/AboutSidebar.jsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
+/**
```
Comment (kevinli-work) [inline src/js/components/about/AboutSidebar.jsx]: This seems almost identical to AgencySidebar. Any reason why we couldn't just use that component and make the modifications necessary to accept CSS class name as a prop (or wrap around child content)?

File: src/_scss/pages/about/header/header.scss (hunk 3 above)
Comment (mab6bf) [reply src/_scss/pages/about/header/header.scss]: Copied CSS from Agency - will remove

File: src/_scss/pages/about/_section.scss (hunk 1 above)
Comment (mab6bf) [reply src/_scss/pages/about/_section.scss]: Was matching to the mock via Inspect mode. Assuming the mock wasn't fully fleshed out, I can just remove that last `rem(13)`.

File: src/_scss/pages/about/_contentWrap.scss (hunk 2 above)
Comment (mab6bf) [reply src/_scss/pages/about/_contentWrap.scss]: Done

File: src/js/components/about/AboutSidebar.jsx (hunk 4 above)
Comment (mab6bf) [reply src/js/components/about/AboutSidebar.jsx]: Done

**Pull request merged**